packages/@internationalized/date/src/queries.ts

```diff
@@ -180,13 +180,30 @@ export function getMinimumDayInMonth(date: AnyCalendarDate) {
   return 1;
 }
 
+const DAY_MAP = {
+  sun: 0,
+  mon: 1,
+  tue: 2,
+  wed: 3,
+  thu: 4,
+  fri: 5,
+  sat: 6
+};
+
+type DayOfWeek = 'sun' | 'mon' | 'tue' | 'wed' | 'thu' | 'fri' | 'sat';
```

Comment:
Should this be in the @react-types package?

Reply:
Similar to the above, I didn't think it was worth exporting, but open to changing if that would be better.